opendkim/opendkim.c

```diff
@@ -10623,6 +10623,7 @@ dkimf_ar_all_sigs(char *hdr, size_t hdrlen, DKIM *dkim,
 		char *result;
 		char *dnssec;
 		char *domain;
+		char *sel;
```

Comment:
I'd prefer "selector" instead of "sel".

Reply:
Shoulda gone with my first instinct which was "selector." :)